python/pyarrow/array.pxi

```diff
@@ -996,15 +996,19 @@ cdef class Array(_PandasConvertible):
         type_format = object.__repr__(self)
         return '{0}\n{1}'.format(type_format, str(self))
 
-    def to_string(self, int indent=0, int window=10):
+    def to_string(self, int indent=0, int window=10,
+                  c_bool skip_new_lines=False):
```

Comment:
We should probably make this new argument keyword-only.

Reply:
👍